server/plugin/utils.go

```diff
@@ -29,19 +30,41 @@ func (p *Plugin) sendEphemeralPost(args *model.CommandArgs, message string, atta
 	)
 }
 
-// Wrapper of p.sendEphemeralPost() to one-line the return statements in all executeCommand functions
+// sendEphemeralResponse is a wrapper of p.sendEphemeralPost() to one-line the return statements in all executeCommand functions
 func (p *Plugin) sendEphemeralResponse(args *model.CommandArgs, message string) *model.CommandResponse {
 	p.sendEphemeralPost(args, message, nil)
 	return &model.CommandResponse{}
 }
 
+// createPost creates a post or logs an error
+func (p *Plugin) createPost(r *model.Post) *model.Post {
+	post, appErr := p.API.CreatePost(r)
+	if appErr != nil {
+		p.API.LogError("Error when creating post", "appError", appErr)
```

Comment:
Should we return `nil` just in case?


```suggestion
		p.API.LogError("Error when creating post", "appError", appErr)
		return nil
```

Reply:
Yes!